spec/rubocop/cop/rspec/undescriptive_literals_description_spec.rb

```diff
@@ -0,0 +1,97 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::UndescriptiveLiteralsDescription, :config do
+  it 'registers an offense when using `describe` with only execute string' do
+    expect_offense(<<~RUBY)
+      describe `time` do
+               ^^^^^^ Description should be descriptive.
+      end
+    RUBY
+  end
+
+  it 'registers an offense when using `context` with only execute string' do
+    expect_offense(<<~RUBY)
+      context `time` do
+              ^^^^^^ Description should be descriptive.
+      end
+    RUBY
+  end
+
+  it 'registers an offense when using `it` with only execute string' do
+    expect_offense(<<~RUBY)
+      it `time` do
+         ^^^^^^ Description should be descriptive.
+      end
+    RUBY
+  end
+
+  it 'registers an offense when using `describe` with only regex' do
+    expect_offense(<<~RUBY)
+      describe /time/ do
+               ^^^^^^ Description should be descriptive.
+      end
+    RUBY
+  end
+
+  it 'registers an offense when using `describe` with only Integer' do
+    expect_offense(<<~RUBY)
+      describe 10000 do
+               ^^^^^ Description should be descriptive.
+      end
+    RUBY
+  end
+
+  it 'does not register an offense when using `describe` with a string' do
+    expect_no_offenses(<<~RUBY)
+      describe '#foo' do
+      end
+    RUBY
+  end
+
+  it 'does not register an offense when using `describe` with a class' do
+    expect_no_offenses(<<~RUBY)
+      describe Foo do
+      end
+    RUBY
+  end
+
+  it 'does not register an offense when using `context` with a string' do
+    expect_no_offenses(<<~RUBY)
+      context 'when foo is bar' do
+      end
+    RUBY
+  end
+
+  it 'does not register an offense when using `it` with a string' do
+    expect_no_offenses(<<~RUBY)
+      it 'does something' do
+      end
+    RUBY
+  end
+
+  it 'does not register an offense when using `describe` with an ' \
+     'interpolation string' do
+    expect_no_offenses(<<~RUBY)
+      describe "foo \#{bar}" do
+      end
+    RUBY
+  end
+
+  it 'does not register an offense when using `describe` with a ' \
+     'heredoc string' do
+    expect_no_offenses(<<~RUBY)
+      describe <<~DESC do
+        foo
+      DESC
+      end
+    RUBY
+  end
+
+  it 'does not register an offense when using `describe` with a string and ' \
+     'execute string' do
+    expect_no_offenses(<<~RUBY)
```

Comment:
What if the doc is a method call or a local var?

Reply:
Added tests for method call and local var. Perhaps these should not be considered offenses. Since we do not know what is in these values or returns value, there will be false positives if they are considered offense.